packages/font-glyphs/src/letter/armenian/other.ptl

```diff
@@ -0,0 +1,121 @@
+$$include '../../meta/macros.ptl'
+
+import [mix linreg clamp fallback] from "@iosevka/util"
+
+glyph-module
+
+glyph-block Letter-Armenian-Other : begin
+	glyph-block-import CommonShapes
+	glyph-block-import Common-Derivatives
+	glyph-block-import Letter-Shared-Shapes : nShoulder nShoulderKnots 
+	glyph-block-import Letter-Shared-Shapes : SerifFrame
+	glyph-block-import Letter-Armenian-Shared-Shapes : ArmHBar LeftHook RightHook
+
+	# Borrowable:
+	## From lower-m:
+	alias 'armn/aybTurned' 0x560 'm'
+	alias 'armn/ayb' 0x561 'turnm'
+	alias 'armn/peh' 0x57A 'turnmLeg'
+
+	## From lower-h:
+	alias 'armn/ho' 0x570 'h'
+
+	## From lower-j:
+	alias 'armn/yi' 0x575 [if SLAB 'dotlessj.flatHookSerifed' 'dotlessj.flatHookSerifless']
+	alias 'armn/yiBar' 0x588 [if SLAB 'dotlessjBar.flatHookSerifed' 'dotlessjBar.flatHookSerifless']
+
+	## From lower-n:
+	alias 'armn/vo' 0x578 'n'
+
+	## From lower-u:
+	alias 'armn/seh' 0x57D 'u'
+
+	## From upper-u:
+	alias 'armn/Seh' 0x54D 'U'
+
+	## From s:
+	alias 'armn/Tiun' 0x54F 'S/sansSerif'
+
+	## From lower-g:
+	alias 'armn/co' 0x581 'cyrl/de.BGR'
+
+	## From phi:
+	alias 'armn/Piur' 0x553 'grek/Phi'
+
+	## From o:
+	alias 'armn/Oh' 0x555 'O'
+	alias 'armn/oh' 0x585 'o'
+
+	do "Ayb" # Upper U Group
```

Comment:
I prefer remove move the empty "do"s here.
You can leave comments in `letter/armenian.ptl`, indicating which letter lives in which file. For example
```ptl
    * Ayb U+0531      => letter/armenian/upper-u-group.ptl
    * Ben U+0532       => letter/armenian/hook-group.ptl
```

Reply:
This whole file is a leftover to be cleaned up anyway, I'll just move all the actual aliases to their corresponding sources and remove this file.